cscs-checks/prgenv/environ_check.py

```diff
@@ -61,6 +69,19 @@ def __init__(self, module_name):
             sn.assert_found(f'{envvar_prefix}_PREFIX', self.stderr),
             sn.assert_found(f'{envvar_prefix}_VERSION', self.stderr)
         ])
+        self.tags = {'production', 'craype'}
+
+
+@rfm.parameterized_test(['cray-fftw'], ['cray-hdf5'], ['cray-hdf5-parallel'],
+                        ['cray-libsci'], ['cray-mpich'], ['cray-netcdf'],
+                        ['cray-netcdf-hdf5parallel'], ['cray-petsc'],
+                        ['cray-petsc-complex-64'], ['cray-python'],
+                        ['cray-R'], ['cray-tpsl'], ['cray-tpsl-64'],
+                        ['cudatoolkit'], ['gcc'], ['papi'], ['pmi'])
+class CrayVariablesCheck(rfm.RunOnlyRegressionTest, CrayVariablesCheckMixin):
+    def __init__(self, module_name):
+        CrayVariablesCheckMixin.__init__(self, module_name)
```

Comment:
This is ugly. We should wait to merge #1699, which it allows you to override the parameter space of the parent class. You can do it also now with the `parameter()` directive, but the syntax changes in #1699.